test/util/spyOnUnsubscribe.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const wrap = require('lodash/wrap');
+
+let unsubscribeSpy;
```

Comment:
The unsubscribeSpy should be declared inside `spyOnUnsubscribe` and note here. The reasoning is that `spyOnUnsubscribe` is a factory for a getter that should have independent states.

Reply:
Got it. Fixed! 😄